test/burials/schema/maximal-test.json

```diff
@@ -0,0 +1,96 @@
+{
+  "data": {
```

Comment:
Can we create unit tests that use this and a minimal test file, like we have for the other forms?